facebook/facebook-business-sdk-codegen
# Fixed and refactored Java SDK samples

## Checklist

- [ ] I've read the [Contributing Guidelines](CONTRIBUTING.md) and the [Code of Conduct](CODE_OF_CONDUCT.md)
- [ ] I've completed the [Contributor License Agreement](https://developers.facebook.com/opensource/cla)

## Pull Request Details

- renamed samples artifact id to facebook-java-business-sdk-samples;
- added the missed package to examples classes;
- refactored the ExampleConfig class (reused the API context from the ExampleConfig class).

## Test Plan

- regenerate the facebook sdk code for java;
- add the facebook-java-business-sdk-samples artifact as maven module;
- check a valid of compile the samples.
